task_list_app/task_list_app/src/main/java/com/jackson/task_list_app/api/controller/TaskController.java

```diff
@@ -24,13 +26,26 @@ public class TaskController {
 
     @GetMapping("/tasks")
     @ResponseStatus(HttpStatus.OK)
-    public List<MyTask> getTask() {
+    public List<Task> getTask() {
         return taskService.getTask();
     }
 
     @PostMapping("/tasks")
-    public ResponseEntity<MyTask> createMyTask(@RequestBody MyTask newMyTask) {
-        MyTask createdMyTask = taskService.createMyTask(newMyTask);
-        return new ResponseEntity<>(createdMyTask, HttpStatus.CREATED);
+    public ResponseEntity<Task> createTask(@RequestBody Task newTask) {
+        Task createdTask = taskService.createTask(newTask);
+        return new ResponseEntity<>(createdTask, HttpStatus.CREATED);
+    }
+
+    @PutMapping("/tasks/{id}") // Adjust the path variable according to your requirements
```

Comment:
Is this comment there from a copy-paste? Is it still providing value?

Comments in the code still need to be maintained. They can get out-of-date and sometimes they can even be misleading. If it is still providing value, please resolve this comment. If not, please edit it or delete it.